### PR TITLE
[Server] Added filters in connections API

### PR DESCRIPTION
**Pull request by Yashsharma1911**
**Notes for Reviewers**

This PR fixes #

Added filter param in connections API

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit dafe8d54c7946f38f1aff2ffd2fa886c1630613b at: https://658c1223e78d1951c4c17e0d--meshery-docs-preview.netlify.app

File: server/handlers/connections_handlers.go [hunk 1]
```diff
@@ -211,9 +212,9 @@ func (h *Handler) GetConnections(w http.ResponseWriter, req *http.Request, prefO
 		order = "updated_at desc"
 	}
 
-	h.log.Debug(fmt.Sprintf("page: %d, page size: %d, search: %s, order: %s", page+1, pageSize, search, order))
+	h.log.Debug(fmt.Sprintf("page: %d, page size: %d, search: %s, order: %s", page+1, pageSize, search, order, filter))
```
Comment (Yashsharma1911) [inline server/handlers/connections_handlers.go:215]:
```suggestion
```

**Comment by l5io** [comment]
🚀 Preview for commit 6233afbe9f693225530b19756e09f143621d23d5 at: https://658c263b727b001c18888b1b--meshery-docs-preview.netlify.app

File: docs/_data/swagger.yml [hunk 2]
```diff
@@ -736,6 +736,8 @@ paths:
                 ```?page={page-number}``` Default page number is 0
 
                 ```?pagesize={pagesize}``` Default pagesize is 10
+
+                ```?filter={filter}``` Filter connections with type or sub_type
```
Comment (MUzairS15) [inline docs/_data/swagger.yml:740]: Did you manually edited this file?

Comment (MUzairS15) [reply docs/_data/swagger.yml:740]: https://github.com/meshery/meshery/pull/9729/files#diff-93e0222a2d4ff5b9d401573f795ea74715141d16731ac18c08af5d109128b1d8R191

Update this comment ad then run `make swagger-build`.

Comment (MUzairS15) [reply docs/_data/swagger.yml:740]: Will you also add an example of the filter param, stating how it can be used?

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

**Comment by Yashsharma1911** [comment]
@MUzairS15 done from side

**Comment by Yashsharma1911** [comment]
// @sudhanshutech

**Comment by Yashsharma1911** [comment]
Moving forward here

**Pull request merged**